source/ContentScript/recorder.ts

```diff
@@ -24,6 +24,7 @@ import {
   ZestStatementElementClick,
   ZestStatementElementSendKeys,
   ZestStatementLaunchBrowser,
+  ZestStatementSwichToFrame,
```

Comment:
There's a typo in "Switch".

Reply:
Nice catch (the name of the element is correct).